src/components/TableCreation.tsx

```diff
@@ -123,7 +123,8 @@ const TableCreation = ({ rows, rowContent, columns, tableId, customTableWidth }:
     return sortedRows;
   }, [rows, sortedColumn, sortDirection]);
 
-  const rowHeight = tableId === 'waiting' || tableId === 'miners' ? 64.8 : 52.813; // row height per table ids, to get table's height
+  const rowHeight =
```

Comment:
this id is needed to take the corresponding row height based on what type of table it is.